addons/im_livechat/static/src/legacy/widgets/livechat_button.js

```diff
@@ -102,6 +101,7 @@ const LivechatButton = Widget.extend({
      */
     _addMessage(data, options) {
         options = Object.assign({}, this.options, options, {
+            default_username: this.messaging.livechatButtonView.defaultUsername,
```

Comment:
You could probably use `this.messaging.livechatButtonView.defaultUsername` directly in `WebsiteLivechatMessage`, so that you can get rid of the options.

Reply:
Indeed, I wasn't sure whether this would imply complications.
I played safe and just turned `LivechatButton` attributes into `LivechatButtonView` fields.
At a glance, this looks safe.

Reply:
In the end, not converted `WebsiteLivechatMessage/defaultUsername` just because relatively unsafe for now.
I much prefer doing it at a later step, and do code cleaning afterwards, as all steps are safe, and globally it's a gain of time. Whereas doing it now is unsafe, so very likely to be a waste of time. Thanks!